### PR TITLE
Remove Legacy Library Projects

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Per https://jira.autodesk.com/browse/DYN-5305, Remove Legacy Library Projects which leverage CEFSharp and MSWebBrowser. There is another task to add unit test for current Library project

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

N/A

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by QilongTang** [comment]
@RobertGlobant20 @reddyashish @zeusongit Any comments?

File: src/Dynamo.All.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.1.32228.430
+# Visual Studio Version 16
```
Comment (reddyashish) [inline src/Dynamo.All.sln:3]: Any reason to downgrade the VS version?

**Review by reddyashish — COMMENTED** [review]
LGTM with one comment.

**Review by RobertGlobant20 — COMMENTED** [review]
@QilongTang there was some files having this line:
[assembly: InternalsVisibleTo("LibraryViewExtensionMSWebBrowser")] or [assembly: InternalsVisibleTo("LibraryViewExtension")]

did you also remove those line?

File: src/Dynamo.All.sln (hunk 1 above)
Comment (QilongTang) [reply src/Dynamo.All.sln:3]: I think this was touched by others who have VS2022, my local VS2019 are still referenced in this case which is `16`

**Comment by mjkkirschner** [comment]
Hey @QilongTang can you verify that the new library actually still builds / contains resources? I thought I remembered that it was sym linked to the resource files and json from the old lib project.

**Comment by QilongTang** [comment]
> Hey @QilongTang can you verify that the new library actually still builds / contains resources? I thought I remembered that it was sym linked to the resource files and json from the old lib project.

Yes, we noticed when you created the MSWebBrowser project, you had an copy step. When we create this new project, we removed the copy step and had all resources moved over

**Comment by QilongTang** [comment]
> @QilongTang there was some files having this lines: [assembly: InternalsVisibleTo("LibraryViewExtensionMSWebBrowser")] or [assembly: InternalsVisibleTo("LibraryViewExtension")]
> 
> did you also remove those line?

Good one, updated

**Comment by QilongTang** [comment]
Passed here: https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/1100/

**Pull request merged**